.mage/dev.go

```diff
@@ -68,8 +69,72 @@ func (Dev) Misspell() error {
 	)
 }
 
+var (
+	devDatabases          = []string{"cockroach", "redis"}
+	devDataDir            = ".dev/data"
+	devDatabaseName       = "ttn_lorawan_dev"
+	devDockerComposeFlags = []string{"-p", "lorawan-stack-dev"}
+)
+
+func dockerComposeFlags(args ...string) []string {
+	return append(devDockerComposeFlags, args...)
+}
+
+func execDC(args ...string) error {
```

Comment:
Probably better to call this `execDockerCompose` in full, since `execDC` isn't obvious in this package.